### ISSUE TITLE
Official Plugin Repository unavailable for 3.34

### ISSUE THREAD

**Issue by benz0li** [qgis/QGIS#55077]
### What is the bug or the crash?

The official Plugin Repository reports as unavailable.

### Steps to reproduce the issue

1. Go to Plugins
2. Click on Manage & Install Plugins
3. Go to Settings Tab
4. Plugin Repositories - Official QGIS Repository is listed as unavailable

### Versions

Not possible because it is a docker build.

```
0.892 Downloading https://plugins.qgis.org/plugins/plugins.xml?qgis=3.34…
0.892 	HTTP Error 404: Not Found
```

See https://gitlab.b-data.ch/jupyterlab/qgis/base/-/jobs/86264#L5447

### Supported QGIS version

- [X] I'm running a supported QGIS version according to [the roadmap](https://www.qgis.org/en/site/getinvolved/development/roadmap.html#release-schedule).

### New profile

- [X] I tried with a new [QGIS profile](https://docs.qgis.org/latest/en/docs/user_manual/introduction/qgis_configuration.html#working-with-user-profiles)

### Additional context

Cross reference: https://github.com/qgis/QGIS/issues/53568

**Comment by benz0li** [comment]
Ping @agiudiceandrea @dimasciput

**Comment by benz0li** [comment]
FYI @jef-n

**Comment by agiudiceandrea** [comment]
Unfortunately it seems this is a recurring issue that happens at almost every new release. I hope someone having admin access rights (maybe @timlinux) can add the missing release version.

**Comment by agiudiceandrea** [comment]
@benz0li, issue fixed: see https://github.com/qgis/QGIS-Django/pull/243#issuecomment-1783207482.

**Comment by Gustry** [comment]
https://plugins.qgis.org/plugins/plugins.xml?qgis=3.34 works now.

**Comment by benz0li** [comment]
@agiudiceandrea @Gustry Mille grazie. Merci beaucoup.

Have a great weekend.

**Comment by ZamirKhurshid** [comment]
I am facing same issue when I go to Plugin> Manage and install plugin the the first window open then I click on ok then it shows error in python plugin repository. how I fix this issue.
![Plugin](https://github.com/qgis/QGIS/assets/108117171/52309d9e-e625-4df9-a3a5-fd1e7da6d3e7)
![Plugin2](https://github.com/qgis/QGIS/assets/108117171/cf29de04-1c04-4e6b-b94f-8c571aaf8eb7)

**Comment by agiudiceandrea** [comment]
@ZamirKhurshid, what is the QGIS version you are using?

**Comment by ZamirKhurshid** [comment]
@agiudiceandrea thannk you for your response I am using 3.28 in my ubuntu system. I have changed it manually as well but it not works. I have noe changed it again but shows same error.
![Uploading SC.png…]()

**Comment by ZamirKhurshid** [comment]
@agiudiceandrea thank you sir for your response. I am using 3.22.4 in mu Ubuntu system. I have changed the manual URL, but shows again same error.
![URL](https://github.com/qgis/QGIS/assets/108117171/2626907c-3165-45de-aa00-7c17963fd31b)
![Version](https://github.com/qgis/QGIS/assets/108117171/c9dec996-7ddb-427c-bc1b-74ba4c99ede7)

**Comment by benz0li** [comment]
> I have changed the manual URL, but shows again same error.

Delete the second URL. It is incorrect.

Only the first URL is correct for QGIS v3.22.4.

**Comment by agiudiceandrea** [comment]
@ZamirKhurshid, as said before, you need to remove the "QGIS Python Plugins Repository" URL-

**Comment by ZamirKhurshid** [comment]
I have deleted the second URL and restart the QGIS but it does not show OSM standard map.
![1](https://github.com/qgis/QGIS/assets/108117171/2047f820-d01c-431d-8001-a6393d8b7934)
![2](https://github.com/qgis/QGIS/assets/108117171/f2f498da-56a9-448a-aaa5-facc5548f802)

**Comment by agiudiceandrea** [comment]
@ZamirKhurshid, the issue about the OSM standard map has nothing to do with this issue report. Moreover QGIS 3.22 is no longer supported: please update it to a currently supported QGIS version and try if the issue does occur also using a new QGIS user profile. If yes, the file a new issue report.

**Comment by ZamirKhurshid** [comment]
@agiudiceandrea Ok sir thank you I will install updated version.

**Comment by ashenafi02** [comment]
Hello Sir, I'm the interface user from Ethiopia. I installed the latest version (3.38.2) and I also faced this challenge.
![Pligins](https://github.com/user-attachments/assets/a310e559-e378-46d5-866e-9b1d9b10fe7f)

**Comment by Gustry** [comment]
@ashenafi02 I think you should check your internet connection.

The link https://plugins.qgis.org/plugins/plugins.xml?qgis=3.38 is working well for me.

**Comment by ashenafi02** [comment]
Thank you. If that is the case,  I'll do so in a strong connection.

**Comment by ssangee123** [comment]
![Image](https://github.com/user-attachments/assets/d665d316-111b-4ad3-aa08-d7738f7ceeff)

I AM USING 3.40 version, still i get unavailable or disconnected. how to fix default plugin issue

**Comment by Gustry** [comment]
@ssangee123 The URL is good for me : https://plugins.qgis.org/plugins/plugins.xml?qgis=3.40

I suggest you check if your connection is OK, does the link work in your web browser ?

Maybe the server was overloaded temporary at this time.

**Comment by ssangee123** [comment]
![Image](https://github.com/user-attachments/assets/04fda861-76b6-4b8f-90ea-82a39902cacb)

i have been facing this issue for couple of weeks. everytime i open plugin it says fetching the repository

**Comment by agiudiceandrea** [comment]
@ssangee123, does your computer connects to Internet through a proxy? If it does, are the correct proxy settings set in QGIS? Is QGIS able to connect to and use any Internet resource like the OpenStreetMap XYZ Tiles layer? Have you tried if the issue still occurs even using a new QGIS user profile?

**Comment by faridcher** [comment]
Go to `Settings > Options > Network > Cache Settings section > Authentication tab` and press `Clear auth connection cache`. Close and reopen QGIS. This solved my problem in QGIS 3.44.

<img width="1415" height="942" alt="Image" src="https://github.com/user-attachments/assets/94b91550-c983-4ee8-aa76-34d6d981d88d" />
